src/app/user/common/user.service.ts

```diff
@@ -16,12 +16,14 @@ export class UserService {
 
   constructor(private http: Http, private jsonp: Jsonp) { }
 
-  // only active users are retrieved
-  getUsers(): Observable<User[]> {
-    const url = userUrl + '/active';
+  // Page data always starts at offset zero (0)
+  // Only active users are retrieved
+  // Returns a JSON object with the data array of Users and totalItems count
+  public getUsers(page: number): Observable<any> {
```

Comment:
since this will be the example for other parts of the site to be "pagified", let's create a model/type, possibly generic

```
interface PagifiedResult <T> {
  totalElements: number,
  data: Array<T> // or T[]
}
```